src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/measure/ActionMeter.kt

```diff
@@ -50,7 +56,13 @@ class ActionMeter private constructor(
     )
 
     @Deprecated(
-        message = "Use ActionMeter.builder instead"
+        message = "Use ActionMeter.Builder instead",
+        replaceWith = ReplaceWith("ActionMeter.Builder(" +
```

Comment:
Not sure if I should include formatting here, but if I put it into one line IntelliJ was doing some weird formatting. (kotlin multiline string did not work here)

Reply:
Nice, you went the extra mile. I'd be fine with a single long line. This is about the content of the replacement. We can leave the callers code style to the callers IDE.